static/app/views/replays/detail/network/index.tsx

```diff
@@ -169,6 +172,42 @@ function NetworkList({
     );
   };
 
+  const handleClick = useCallback(() => {
+    const frame = getNextReplayFrame({
+      frames: items,
+      targetOffsetMs: currentTime,
+      allowExact: true,
+    });
+    const frameIndex = items.findIndex(spanFrame => frame === spanFrame);
+    // index needs to be at least 1 to be a valid index to jump to
+    const index = frameIndex < 1 ? 1 : frameIndex;
+    setScrollToRow(index);
+  }, [setScrollToRow, currentTime, items]);
+
+  function indexAtCurrentTime() {
+    const frame = getNextReplayFrame({
+      frames: items,
+      targetOffsetMs: currentTime,
+      allowExact: true,
+    });
+    const frameIndex = items.findIndex(spanFrame => frame === spanFrame);
+    // frameIndex is -1 when the page is loading, so the Jump Up Button appears until the page finishes loading in
+    const index = frameIndex === -1 ? 0 : frameIndex;
+    return index;
+  }
+
+  function pixelsToRow(pixels) {
+    return Math.floor(pixels / BODY_HEIGHT);
+  }
+
+  const showJumpDownButton = () => {
+    return indexAtCurrentTime() > visibleRange[1];
+  };
+
+  const showJumpUpButton = () => {
+    return indexAtCurrentTime() < visibleRange[0];
+  };
```

Comment:
We could probably turn these functions into variables, and then we're only need to call `indexAtCurrentTime()` one time.
That would be a little perf improvement because `indexAtCurrentTime()` is the expensive method here, it's finding the frame and finding the index of it, so if we can call it once that's better.

```suggestion
  const currentIndex = indexAtCurrentTime();
  const showJumpDownButton = currentIndex > visibleRange[1];
  const showJumpUpButton = currentIndex < visibleRange[0];
```